test/helpers/data/aggregated-data-helper.js

```diff
@@ -97,10 +97,17 @@ module.exports.addWorkloadCapacitiesForOffenderManager = function () {
   var promise = knex('workload_points').returning('id').insert(defaultWorkloadPoints)
     .then(function (ids) {
       inserts.push({table: 'workload_points', id: ids[0]})
-      return knex('offender_manager_type').returning('id').insert({description: 'test'})
+
+      var offenderManagerTypes = [
+        { grade_code: 'PO' },
+        { grade_code: 'PSO' }
+      ]
+      return knex('offender_manager_type').returning('id').insert(offenderManagerTypes)
     })
     .then(function (ids) {
-      inserts.push({ table: 'offender_manager_type', id: ids[0] })
+      ids.forEach((id) => {
+        inserts.push({table: 'offender_manager_type', id: id})
```

Comment:
addRegion only inserts one record from the given array. So the inserts array will be all inserted. Does it make sense to keep it as it was?

Reply:
Before it was only adding one offender manager type, which wasn't a problem as we were defining grade_code in the offender_manager rows. Now we are taking grade_code from the offender_manager_type table so need more test data. 

Region adds one, but in the method used for the caseload tests, we specifically add PO and PSO offender managers, so both need to be in the offender_manager_type data